bindata/configmaps/console-managed-cluster-ingress-cert-configmap.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-config-managed
```

Since this is a config specific to the console, I think this should be created in the `openshift-console` namespace. 
```suggestion
  namespace: openshift-console
```